api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -42,6 +41,7 @@
   private RdrExportDao rdrExportDao;
   private WorkspaceDao workspaceDao;
   private UserDao userDao;
+  private WorkspaceService workspacesApi;
```

Comment:
Naming nit: for consistency w/ the class name, prefer 'workspaceService' as the member variable name.